<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primesc</class>
 <widget class="QWidget" name="primesc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Tune_but">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20pt &quot;Ubuntu&quot;;

border-radius:20px;
border:none;
font:bold;border:3px solid black;</string>
   </property>
   <property name="text">
    <string>TUNE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prime1_but">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20pt &quot;Ubuntu&quot;;

border-radius:20px;
border:none;
font:bold;border:3px solid black;</string>
   </property>
   <property name="text">
    <string>PRIME</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clean_but">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>470</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20pt &quot;Ubuntu&quot;;

border-radius:20px;
border:none;
font:bold;
border:3px solid black;</string>
   </property>
   <property name="text">
    <string>CLEAN</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>801</width>
     <height>711</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(153, 193, 241);</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab_6">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 193, 241);</string>
    </property>
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QLabel" name="process_lab_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:25pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
border-radius:20px;
border:none;font:bold;</string>
     </property>
     <property name="text">
      <string>     PROCESS </string>
     </property>
    </widget>
    <widget class="QPushButton" name="start_prime_but_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>410</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
border-radius:20px;
border:none;font:bold;</string>
     </property>
     <property name="text">
      <string>START PRIME</string>
     </property>
    </widget>
    <widget class="QLabel" name="arrow1_lab_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/downn.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="arrow2_lab_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>190</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/downn.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="arrow3_lab_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/downn.png);
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="start_check_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>501</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu Mono&quot;;
</string>
     </property>
     <property name="text">
      <string>Start Irrigation Valve</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="wait_Check_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu Mono&quot;;
</string>
     </property>
     <property name="text">
      <string>Wait For Chamber To Be Filled</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="motor_Check_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>301</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu Mono&quot;;
</string>
     </property>
     <property name="text">
      <string>Start Motor</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="done_Check_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu Mono&quot;;
</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>561</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/next.png);
border:none;</string>
     </property>
     <property name="text">
      <string>bbbb</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 193, 241);</string>
    </property>
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QLabel" name="handpice_img_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>-50</y>
       <width>611</width>
       <height>361</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/hand.png);
border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Start_tune_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>340</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
border-radius:20px;
border:none;
font:bold;</string>
     </property>
     <property name="text">
      <string>START TUNING </string>
     </property>
    </widget>
    <widget class="QLabel" name="progress_lab_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>501</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="tick_lab_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>260</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="initial_lab_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/notconnected.png);
background-color:transparent;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/next.png);
border:none;</string>
     </property>
     <property name="text">
      <string>bbbbbb</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>460</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/settings.png);
border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <property name="styleSheet">
     <string notr="true">background-color: rgb(153, 193, 241);</string>
    </property>
    <attribute name="title">
     <string>Page</string>
    </attribute>
    <widget class="QLabel" name="warning_lab_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:25pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
border-radius:20px;
border:none;
font:bold;</string>
     </property>
     <property name="text">
      <string>   BEFORE CLEANING</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="Instruction_lab_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>571</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none;
background-color: rgb(222, 221, 218);</string>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;1.Disconnect the Irrigation/Aspiration connectors from handpiece&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;2.Connect the irrigation,aspiration connectors together&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;3.Disconnect the saline connection from irrigation.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;4.Disconnect the tubing from irriagtion valve.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:12pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;5.dip the irrigation connector in distilled water bowl&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="begin_clean_but_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>440</y>
       <width>291</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font:20pt &quot;Ubuntu&quot;;
background-color: rgb(36, 31, 49);
color: rgb(255, 255, 255);
font:bold;
border-radius:20px;
border:none;</string>
     </property>
     <property name="text">
      <string>Begin Clean</string>
     </property>
    </widget>
    <widget class="QLabel" name="warning_img_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/warning.png);
BACKGROUND-COLOR:TRANSPARENT;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>511</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 20pt &quot;Ubuntu&quot;;
font:bold;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string> Instructions To Be Followed:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>410</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/next.png);
border:none;
background-color:transparent;</string>
     </property>
     <property name="text">
      <string>bbbbbb</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
